test/e2e/manifest_test.go

```diff
@@ -297,7 +304,15 @@ var _ = Describe("Podman manifest", func() {
 		registryOptions := &podmanRegistry.Options{
 			Image: "docker-archive:" + imageTarPath(REGISTRY_IMAGE),
 		}
-		registry, err := podmanRegistry.StartWithOptions(registryOptions)
+
+		// registry script invokes $PODMAN; make sure we define that
+		// so it can use our same networking options.
+		opts := strings.Join(podmanTest.MakeOptions(nil, false, false), " ")
```

Comment:
Do we have to worry about options containing a white space, should we escape the options?